jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -330,6 +354,16 @@ protected void checkJibVersion() throws MojoExecutionException {
     MojoCommon.checkJibVersion(descriptor);
   }
 
+  /**
+   * Gets the specified platforms.
+   *
+   * @return the specified platforms
+   */
+  @Nullable
```

Comment:
At the current state, I don't think this method can ever return null. In that case, we should remove `@Nullable`.

Reply:
Thats true .Done.